### PR TITLE
Maslan

### PR DESCRIPTION
**Pull request by Maslan22**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -75,6 +75,24 @@ function App() {
             </>
           }
         />
+        <Route
```
Comment (Lukada-taiya) [inline src/App.tsx:78]: the root path - "/" should take you to the login page not the dashboard

Comment (Lukada-taiya) [reply src/App.tsx:78]: The routes should be modified to reflect that

File: src/components/Header/index.tsx [hunk 2]
```diff
@@ -56,8 +59,8 @@ const Header = (props: {
           </button>
           {/* <!-- Hamburger Toggle BTN --> */}
 
-          <Link className="block flex-shrink-0 lg:hidden" to="/">
-            <img src={LogoIcon} alt="Logo" />
```
Comment (Lukada-taiya) [inline src/components/Header/index.tsx:60]: for all logos, add check to switch suitable logo if in dark mode

Comment (Maslan22) [reply src/components/Header/index.tsx:60]: done

**Pull request merged**